### DISCUSSION TITLE
Implement Attribute accessors

### DISCUSSION THREAD

**Discussion by ivandokov** [livewire/livewire#4478] [3. Feature ideas]
Do you have plans to implement Attribute for properties like Taylor did for attributes accessors in this PR https://github.com/laravel/framework/pull/40022

**Comment by joshhanley** [comment]
@ivandokov I'm not sure what you mean, Livewire is built on Laravel, so you should have access to them? If there is something extra, can you explain what you are after, ideally with some example code?

Thanks!

**Reply by ivandokov** [reply]
I meant Computed Properties https://laravel-livewire.com/docs/2.x/properties#computed-properties
Not sure why I wrote only "properties".

The idea is instead of naming the computed property function `get<PropName>Property()` to use `Attribute` return type as Taylor implemented for accessors and mutators. Myabe another return type `Property` can be added.

```php
// old
public function getPostProperty()
{
    return Post::find($this->postId);
}
```


```php
// new
public function post(): Attribute
{
    return new Attribute(
        get: fn () => Post::find($this->postId),
    );
}
```

**Reply by joshhanley** [reply]
@ivandokov ah right yeah I understand now 🙂
